packages/suite-native-settings/src/navigation/SettingsStackNavigator.tsx

```diff
@@ -1,25 +1,51 @@
 import React from 'react';
-import { createStackNavigator } from '@react-navigation/stack';
-import { SettingsScreen } from '../screens/SettingsScreen';
-import { SettingsDetailScreen } from '../screens/SettingsDetailScreen';
+import { createStackNavigator, TransitionPresets } from '@react-navigation/stack';
+
+import { stackNavigationOptionsConfig } from '@suite-native/navigation-root';
 import { SettingsStackRoutes, SettingsStackParamList } from './routes';
+import { SettingsScreen } from '../screens/SettingsScreen';
+import { SettingsLocalisationScreen } from '../screens/SettingsLocalisationScreen';
+import { SettingsLabelingScreen } from '../screens/SettingsLabelingScreen';
+import { SettingsAdvancedScreen } from '../screens/SettingsAdvancedScreen';
 
-const SettingsStack = createStackNavigator<SettingsStackParamList>();
+export const SettingsStack = createStackNavigator<SettingsStackParamList>();
 
 export const SettingsStackNavigator = () => (
     <SettingsStack.Navigator
         initialRouteName={SettingsStackRoutes.Settings}
-        screenOptions={{ headerShown: false, gestureEnabled: true, gestureDirection: 'horizontal' }}
+        screenOptions={stackNavigationOptionsConfig}
     >
         <SettingsStack.Screen
-            options={{ title: 'Settings' }}
+            options={{
+                title: SettingsStackRoutes.Settings,
+                ...TransitionPresets.SlideFromRightIOS,
```

Comment:
should be moved to global stack config

Reply:
[8c23b9f](https://github.com/trezor/trezor-suite/pull/5617/commits/8c23b9f9cafd67684e4e1e0e3c5c399d53c6bc34)